.github/workflows/bi-weekly-UX-WG-meeting-reminder.yaml

```diff
@@ -0,0 +1,37 @@
+# Copyright 2024 The Knative Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: 'Bi-weekly UX WG meeting reminder'
+
+on:
+  workflow_dispatch:
+  schedule: ## At 12:55 AM UTC on every other Thursdays
+    - cron: 55 12 * * 4
```

Comment:
Could we set this to go out on Wednesdays instead? That way people have a bit of a heads up the day before

Reply:
Although I think a reminder that is sent 5 min before the meeting is helpful too, if doing both (1 day reminder and 5 min reminder) will be too many notifications.